Update azurerm_eventhub_namespace with optional dedicated_cluster_id (#7548)
terraform-providers/terraform-provider-azurerm: website/docs/r/eventhub_namespace.html.markdown

* `auto_inflate_enabled` - (Optional) Is Auto Inflate enabled for the EventHub Namespace?

> * `dedicated_cluster_id` - (Optional) Specifies if the EventHub Namespace should be deployed on a Dedicated EventHubs Cluster. Changing this forces a new resource to be created.

Reviewer: ```suggestion
* `dedicated_cluster_id` - (Optional) Specifies the ID of the EventHub Dedicated Cluster where this Namespace should created. Changing this forces a new resource to be created.
```
